web-client/src/lib.rs

```diff
@@ -513,21 +501,62 @@ impl Client {
         &self,
         addresses: &AddressAnyArrayType,
     ) -> Result<PlainAccountArrayType, JsError> {
-        // Unpack the array of addresses
-        let js_value: &JsValue = addresses.unchecked_ref();
-        let array: &Array = js_value
-            .dyn_ref()
-            .ok_or_else(|| JsError::new("`addresses` must be an array"))?;
-        let mut addresses = Vec::<_>::with_capacity(array.length().try_into()?);
-        for any in array.iter() {
-            let address = Address::from_any(&any.into())?;
-            addresses.push(address);
-        }
-
+        let addresses = Client::unpack_addresses(addresses)?;
         let plain_accounts = self.get_plain_accounts(addresses).await?;
         Ok(serde_wasm_bindgen::to_value(&plain_accounts)?.into())
     }
 
+    /// Fetches the staker for the provided address from the network.
+    ///
+    /// Throws if the address cannot be parsed and on network errors.
```

Comment:
Throws an error if the address

Reply:
All docs are like this, I think it's correct.